CHANGELOG.md

```diff
@@ -20,6 +23,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   value can be None, default rabbitmq broker, or redis for a redis
   local broker.
 - Missing doc options for run and run-workers.
+- Add port to rabbitmq config options.
+- Updated docker docs.
```

Comment:
This is more of a `Changed` item in my opinion

Reply:
Fixed